src/EagleSabi.Coordinator.Domain/Context/Round/Events.cs

```diff
@@ -0,0 +1,19 @@
+﻿using EagleSabi.Coordinator.Domain.Context.Round.Records;
+using EagleSabi.Infrastructure.Common.Abstractions.EventSourcing.Models;
+using NBitcoin;
+using WalletWasabi.Crypto;
+
+namespace EagleSabi.Coordinator.Domain.Context.Round;
+
+public record RoundStartedEvent(RoundParameters RoundParameters) : IEvent, IRoundClientEvent;
+public record InputRegisteredEvent(Guid AliceSecret, Coin Coin, OwnershipProof OwnershipProof) : IEvent;
+public record InputUnregistered(OutPoint AliceOutPoint) : IEvent;
+public record InputsConnectionConfirmationStartedEvent() : IEvent, IRoundClientEvent;
+public record InputConnectionConfirmedEvent(Coin Coin, OwnershipProof OwnershipProof) : IEvent, IRoundClientEvent;
+public record OutputRegistrationStartedEvent() : IEvent, IRoundClientEvent;
+public record OutputRegisteredEvent(Script Script, long CredentialAmount) : IEvent, IRoundClientEvent;
+public record SigningStartedEvent() : IEvent, IRoundClientEvent;
+public record InputReadyToSignEvent(OutPoint AliceOutPoint) : IEvent;
+public record SignatureAddedEvent(OutPoint AliceOutPoint, WitScript WitScript) : IEvent, IRoundClientEvent;
```

Comment:
nit: consider changing this because `TransactionSignatureRequest` contains an index, so specifying the event in terms of the outpoint means converting from index to outpoint and then back to index later in the aggregate's `Apply` method

````suggestion
public record SignatureAddedEvent(int InputIndex, WitScript WitScript) : IEvent, IRoundClientEvent;
```

Reply:
delay until later